.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
you might want to require `rails_helper` instead, and avoid the `require 'rails_helper'` in your test file.

Reply:
`rails_helper` includes `spec_helper`, you don't need both

Reply:
bump ^^